packages/api/src/WithServerEvents.tsx

```diff
@@ -0,0 +1,241 @@
+/* tslint:disable */
```

Comment:
Just remember to remove this eventually :)

Reply:
Yep, this is the legacy code, surprisingly removing this didn't turn up as many lint errors as I'd expect :-)